genie-common/src/main/java/com/netflix/genie/common/dto/v4/CommandRequest.java

```diff
@@ -56,12 +56,20 @@
         message = "The minimum amount of memory if desired is 1 MB. Probably should be much more than that"
     )
     private final Integer memory;
+    @Min(
+        value = 1,
```

Comment:
Why not set a more reasonable min, like 100 (still low). Scheduling a check every millisecond could really do some damage...

Reply:
I agree but I left it the same for backwards compatibility for now